### PR TITLE
[#9293] Changed isWhiteSpace

### LINKED ISSUE #9293: Potential NullPointerException
**Steps to reproduce**
Pass null to isWhiteSpace function

**Expected behaviour**
return boolean value

**Actual behaviour**
throws NullPointerException

### PR DESCRIPTION
**Pull request by aravinth2094**
Fixes #9293

### CODE REVIEW

**Comment by aravinth2094** [comment]
Changed the subject

**Comment by aravinth2094** [comment]
Fixes #9293

File: src/main/java/teammates/common/util/StringHelper.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public static String generateStringOfLength(int length, char character) {
     }
 
     public static boolean isWhiteSpace(String string) {
-        return string.trim().isEmpty();
+        return isEmpty(trimIfNotNull(string));
```
Comment (IAmPramod) [inline src/main/java/teammates/common/util/StringHelper.java]: You need to modify the test as well to fix build

**Comment by xpdavid** [comment]
PR on hold until the issue is clarified.

**Comment by xpdavid** [comment]
Closed as discussed.

**Pull request closed without merging**